libcloud/test/common/test_openstack_identity.py

```diff
@@ -790,14 +788,16 @@ def _v3_OS_FEDERATION_identity_providers_idp_protocols_oidc_auth(self, method, u
     def _v3_OS_FEDERATION_projects(self, method, url, body, headers):
         if method == 'GET':
             # get user projects
-            body = json.dumps({"projects": [{"id": "project_id"}]})
+            body = json.dumps({"projects": [{"id": "project_id", "name": "project_name"},
+                                            {"id": "project_id2", "name": "project_name2"}]})
             return (httplib.OK, body, self.json_content_headers, httplib.responses[httplib.OK])
         raise NotImplementedError()
 
     def _v3_auth_projects(self, method, url, body, headers):
         if method == 'GET':
             # get user projects
-            body = json.dumps({"projects": [{"id": "project_id"}]})
+            body = json.dumps({"projects": [{"id": "project_id", "name": "project_name"},
```

Comment:
It would also be good to add a test case for scenario where invalid project id is specified.